app/scripts/lib/setupSentry.test.js

```diff
@@ -239,3 +240,142 @@ describe('Setup Sentry', () => {
     });
   });
 });
+
+// Create a mock implementation of the function to test
```

Comment:
Q: why? Then you are only testing the mock?

Reply:
Removed ! was a bad test
- 7b1f71740541cb39f1b383f90887e9c80ee70fad
- 7c2bf713b5ff40c636650b4a9b6695dfdd642553